backend/doi/link_header.go

```diff
@@ -0,0 +1,75 @@
+package doi
+
+import (
+	"regexp"
+	"strings"
+)
+
+var linkRegex = regexp.MustCompile(`^<(.+)>$`)
+var valueRegex = regexp.MustCompile(`^"(.+)"$`)
+
+// headerLink represents a link as presented in HTTP headers
+// MDN Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/headerLink
```

Comment:
When I tried this link it did not really work. I think you mean https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Link?